[nrfconnect] Added device type related kconfig options (#53)
kkasperczyk-no/connectedhomeip: examples/lighting-app/nrfconnect/README.md

example `nrf52840dk_nrf52840`), edit the `pm_static.yml` file located in the
`configuration/build-target/` directory.

### Building with pre-commissioning device identification features support

To enable the desire features you need to configure the proper Kconfig options.
See [Configuring the example](#configuring-the-example) to read how to configure example's options.

#### Rotating Device Id

You can enable the rotating device id feature by setting the `CONFIG_CHIP_ROTATING_DEVICE_ID=y` Kconfig option.

#### Commissionable Device Type

> You can enable the commissionable device type feature by setting the `CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y` Kconfig option.
> To configure the Matter Device Type Identifier value to the one matching the used device type you need to set `CONFIG_CHIP_DEVICE_TYPE` option. For example the Dimmable Light device type value is equal to `257 (0x0101)`.

Reviewer: ```suggestion
You can enable the commissionable device type feature with the `CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y` Kconfig option.
To configure the Matter device type identifier value to the one matching the used device type, set `CONFIG_CHIP_DEVICE_TYPE` option. For example, the Dimmable Light device type value is equal to `257 (0x0101)`.
```
